experimental/framework/benchmark/package.json

```diff
@@ -0,0 +1,64 @@
+{
+	"name": "@fluid-experimental/benchmark",
```

Comment:
The intention is for it to be generally useful for benchmarking anything you can put in a mocha test.

Since it uses benchmark.js its quite good at micro benchmarks (I've used it for comparing single digit nanosecond times to compare JavaScript vs native function call overhead for example), but also works fine all the way up to application integration test level performance, even async code.

Its even possible to use it with test frameworks other than mocha (we have used with with jest as well), but I'm only including a MochaReporter and runner (other frameworks would need different runners and reporters).

Its Json output is suitable for consumption by CI, and can be used with tools like PowerBI to plot performance over time, but it also has human friendly output for manual use.

Overall I think it's quite general, and expect it can meet the general benchmarking needs of FluidFramework. If it doesn't, it could be extended to do so. That way we can keep a single centralized package for consistency, since having consistent data collection and output is important for tooling/CI integration.

I'll personally be writing benchmarks for experimental tree which I'll be including in our app's internal CI's automated benchmarking, and probably port bubble bench to use it. I won't be writing other benchmarks with it, but I was thinking if it turns out to work well with experimental tree, and anyone else is interested in using it, we would promote it from experimental.